Documentation/dev-guide/interacting_v3.md

```diff
@@ -181,6 +192,41 @@ $ etcdctl del foo foo9
 2 # two keys are deleted
 ```
 
+Here is the command to delete key `zoo` with the deleted key value pair returned:
+
+```bash
+$ etcdctl del --prev-kv zoo 
+1   # one key is deleted
+zoo # deleted key
+val # deleted value
+```
+
+Here is the command to delete keys having prefix as `zoo`:
+
+```bash
+$ etcdctl del --prefix zoo 
+2 # two keys are deleted
+```
+
+## Delete keys which are greater than or equal to the byte value of the specified key
+
+Applications may want to delete keys which are greater than or equal to the byte value of the specified key.
+
+Suppose an etcd cluster already has the following keys:
+
+``` bash
```

Comment:
@gyuho Sorry i had mis-understood. you were referring to bash. Will do that for sure.